Scripts/Services/Discovery/V2/Model/QueryAggregation.cs

```diff
@@ -0,0 +1,34 @@
+/**
+* Copyright 2018, 2019 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using Newtonsoft.Json;
+
+namespace IBM.Watson.Discovery.V2.Model
+{
+    /// <summary>
+    /// An abstract aggregation type produced by Discovery to analyze the input provided.
+    /// </summary>
+    public class QueryAggregation
+    {
+        /// <summary>
+        /// The type of aggregation command used. Options include: term, histogram, timeslice, nested, filter, min, max,
+        /// sum, average, unique_count, and top_hits.
+        /// </summary>
+        [JsonProperty("type", NullValueHandling = NullValueHandling.Ignore)]
+        public string Type { get; set; }
+    }
+}
```

Comment:
There needs to be handwitten change for query aggregation for disco v1 and v2. We need to figure out what this looks like in Unity. In .NET it looks like [this](https://github.com/watson-developer-cloud/dotnet-standard-sdk/blob/7749-november-2019-release/src/IBM.Watson.Discovery.v1/Model/QueryAggregation.cs)